src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -7,9 +7,14 @@
     <services>
         <defaults public="false" />
 
+        <service id="controller_layout.generic" class="Symfony\Component\HttpKernel\Controller\Layout\GenericControllerLayout">
```

Comment:
Could probably be the FQCN as ID, though I'm not sure if this is default for the Symfony Core. If not, I'd like to see an alias for autowiring purposes (perhaps on the interface as well if this is the default).

Reply:
There are no FQCN identifiers in this class, neither did I. I think this is a step for further improvements outside of the scope of this PR